### PR TITLE
v2.10.1

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: neo/SmartContract/Helper.cs [hunk 1]
```diff
@@ -115,7 +115,8 @@ internal static bool VerifyWitnesses(this IVerifiable verifiable, Snapshot snaps
                 {
                     engine.LoadScript(verification);
                     engine.LoadScript(verifiable.Witnesses[i].InvocationScript);
-                    if (!engine.Execute()) return false;
```
Comment (shargon) [inline neo/SmartContract/Helper.cs]: Maybe we could preserve the boolean return

**Comment by shargon** [comment]
We have a similar problem with travis in other pull request.

**Comment by erikzhang** [comment]
There is no problem with travis. We are using daily build of dependency. That's why these PR are drafts. They should not be merged before change to stable dependency.

**Comment by ixje** [comment]
Don't forget to include https://github.com/neo-project/neo/pull/673#discussion_r271302940 before release. @erikzhang should I do a PR or will you push this yourself?

**Comment by erikzhang** [comment]
Make a PR please.

**Pull request merged**